Documentation/cmdref/cilium-operator-alibabacloud.md

```diff
@@ -58,7 +58,7 @@ cilium-operator-alibabacloud [flags]
       --enable-node-ipam                                     Enable Node IPAM
       --enable-node-selector-labels                          Enable use of node label based identity
       --enable-policy string                                 Enable policy enforcement (default "default")
-      --enable-policy-secrets-sync                           Enables fan-in TLS secrets sync from multiple namespaces to singular namespace (specified by tls-interception-secrets-namespace flag)
+      --enable-policy-secrets-sync                           Enables fan-in TLS secrets sync from multiple namespaces to singular namespace (specified by policy-secrets-namespace flag)
```

Comment:
Is the flag renaming a breaking change? was this released at some point?

Reply:
Nope, the flag name was never part of a merged commit. Likely changed during development and not adjusted.